pkg/manual/manual.go

```diff
@@ -4,11 +4,62 @@
 
 package manual
```

Comment:
I'm afraid that `pkg/manual` with cgo might make TiDB depend on gcc and the cgo part of this file was moved once at https://github.com/pingcap/br/pull/525.

Also cc @glorv, is remove cgo still acceptable for now? Or we may need to move the cgo part to another package?

Reply:
It seems we don't need `manual`  package anymore. I have removed the entire `manual`  package. @glorv PTAL